docs/api/window-open.md

```diff
@@ -73,6 +73,11 @@ creating the window. Note that this is more powerful than passing options
 through the feature string, as the renderer has more limited privileges in
 deciding security preferences than the main process.
 
+In addition to passing in `action` and `overrideBrowserWindowOptions`,
+`closeWithOpener` can be passed like: `{ action: 'allow', closeWithOpener: false,
+overrideBrowserWindowOptions: { ... } }`. If set to `false`, child windows will not
+be closed when the opener window closes. The default value is `true`.
```

Comment:
Hm, this is a little unclear, I think it could be interpreted as either "this window will stay open when its opener closes" or as "children of this window will stay open when this window closes". Could you clarify this documentation?

Reply:
"this window will stay open when its opener closes" This would be the correct interpretation.
I will adapt the docs once we agree on the property name.

Reply:
"generally preferable to have the default of a boolean be false". That's a good point
`outlivesParent`, `remainsWhenParentCloses`. I'd like to stay away from the term "parent" since it can be easily confused with parent in the sense of parent-child reltionships, i.e. when setting the parent property during window creation or when using browserWindow.setParentWindow. That's why I went with "opener" instead to put the focus on "window.open" logic. (Also fits with the window.opener property) So I'd suggest to either change to `outlivesOpener` or `remainsWhenOpenerCloses`

Reply:
Let's go with `outlivesOpener`. No ambiguous terms, doesn't have both `open` and `close` in it.

Reply:
Changes are pushed and tested, all looking good, please have a look